src/components/loading/Loading.tsx

```diff
@@ -94,7 +94,7 @@ const Loading = () => {
           Making Image...
         </Typography>
       </div>
-      <Stacks ref={targetRef} selecteds={state} color={color as BgColorOption['color']} />
+      <Stacks ref={targetRef} selecteds={state} color={color as BgColorOption} />
```

Comment:
This part became more concise by making BgColorOption a type that did not need to be declared as an interface.